src/mlpack/core/optimizers/ada_delta/ada_delta_update.hpp

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file ada_delta_update.hpp
+ * @author Abhinav Moudgil
```

Comment:
Can you add ``@author Vasanth Kalingeri`` since he basically implemented the ada delta policy.